### DISCUSSION TITLE
Thank you Tyrrrz!!!!

### DISCUSSION THREAD

**Discussion by devanteweary** [Tyrrrz/YoutubeDownloader#311] [General]
Just wanted to take the time to say thank you for this great tool.

I love it because tools like this are a slap in the face of these shady business that make bloated, spyware infested programs.

This is simple, looks good, does the job, easy to use. I love it man, thank you! Wish there was a donate (unless I missed it!)

**Comment by Tyrrrz** [comment]
You're welcome!

> Wish there was a donate (unless I missed it!)

You can find different donation options here: https://tyrrrz.me/donate
